### DISCUSSION TITLE
Can we streamline by using the excel spreadsheet more?

### DISCUSSION THREAD

**Discussion by johnphd** [jepegit/cellpy#214] [General]
Out of the box, cellpy calculates and exports data based on capacity.   The mass, area, and so on are in the excel sheet.  To get areal capacity, one needs to enter the mass and later cancel it out, 

`cellname="cell_name"`
`area=1.767`
`cycles=cycle_numbers=[1,2,3,4,5]` 

`mass=0.026` 
`area=1.767# in cm^2 `  

this is also present in the excel spreadsheet for both working and reference/counter electrode.

Then, to get the capacity per area we have 

`cell_data=b.experiment.data[cellname]`
 `capacity_voltage=cell_data.get_cap(cycle_numbers, label_cycle_number = True,method = "forth-and-forth")` 
`capacity_voltage`

`capacity_voltage["capacity_area"]= capacity_voltage["capacity"] *mass/area` 

This clearly has a value for mass already and capacity, probably from the spreadsheet, and mAh/g can be plotted without entering mass.  However it is required to manually enter the mass, then later cancel it out in order to get areal capacity.  This has consumed a lot of time already, and I cannot find a way around it although I think once I was close.

**Comment by jepegit** [comment]
Hi @johnphd. Working on this now. It will be a "breaking change", so I guess the feature will be implemented in v.0.5 first.
